.github/workflows/bump-homebrew-formula.yml

```diff
@@ -0,0 +1,34 @@
+# Automatically bump Homebrew formula after each release
+# https://github.com/mislav/bump-homebrew-formula-action
+
+# Controls when the action will run. Triggers the workflow when new tags are pushed
+on:
+  push:
+    tags: '*'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+
+  # This workflow contains a single job called "homebrew"
+  homebrew:
+    name: Bump Homebrew formula
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v1.6
+        if: "!contains(github.ref, '-')" # skip prereleases
+        with:
+          formula-name: dita-ot
+          homebrew-tap: Homebrew/homebrew-core
+          base-branch: master
+          download-url: https://github.com/dita-ot/dita-ot/releases/download/3.4/dita-ot-3.4.zip
```

Comment:
This needs to be fixed. The `download-url` should have the correct version numbers. If we omit it, the action will attempt to [generate](https://github.com/mislav/bump-homebrew-formula-action/blob/master/src/main.ts#L9) the URL but it will not work correctly in our case.

Reply:
Fixed